syntax = "proto2";

package tcash.elect.protobuf;

message VhostMessage {
	optional uint32 type = 1;
	optional bytes data = 2;
	optional uint32 cb_type = 3;
	optional bytes node_id = 4;
}
